scheme/app/DoctrineMigrations/Version20190410082735.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20190410082735 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE BillableCalls ADD direction VARCHAR(255) DEFAULT \'outbound\' COMMENT \'[enum:inbound|outbound]\'');
```

Comment:
255 seems a bit excessive for a 10 character field. Do not know if mysql optimizes this.

Reply:
Varchar columns are variable-length strings. I would not worry about this